morebits.js

```diff
@@ -1203,14 +1190,10 @@ Morebits.array = {
 		if (typeof size !== 'number' || size <= 0) { // pretty impossible to do anything :)
```

Comment:
Hmm sorry didn't notice this comment earlier and I see you've merged it, but wanted to note that this wouldn't have been a good idea. By doing `return [ arr ];` it would be returning an array containing nothing but a reference to the original array!! If the original array is modified, then the chunked array also gets modified, which would be unexpected behavior leading to insidious bugs - it would have meant modifying the original array was ok when size < arr.length but not when size >= arr.length. 

By using `slice` it's ensured that the returned array is a different one. 

The handling was flawed to begin with and IMO this is another place where it's better to just throw.